release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -375,6 +375,44 @@ Pods created using the macvlan CNI plugin, where the IP address management CNI p
+
+[id="ocp-4-15-storage-non-graceful-node-shutdown"]
+==== Detach CSI volumes after non-graceful node shutdown is generally available
+Starting with {product-title} 4.13, Container Storage Interface (CSI) drivers can automatically detach volumes when a node goes down non-gracefully as a Technology Preview feature. When a non-graceful node shutdown occurs, you can then manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is now generally available.
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-vol-detach-non-graceful-shutdown.adoc[Detach CSI volumes after non-graceful node shutdown].
+
+[id="ocp-4-15-storage-gcp-filestore-shared-vpc"]
+==== Shared VPC is supported for the GCP Filestore CSI Driver Operator as generally available
+Shared virtual private cloud (VPC) for the Google Compute Platform (GCP) Container Storage Interface (CSI) Driver Operator is now supported as a generally available feature. Shared VPC simplifies network management, allows consistent network policies, and provides a centralized view of network resources.
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-google-cloud-file-create-sc_persistent-storage-csi-google-cloud-file[Creating a storage class for GCP Filestore Storage].
+
+[id="ocp-4-15-storage-ibm-vpc-byok"]
+==== User-Managed encryption supports IBM VPC Block storage as generally available
+The user-managed encryption feature allows you to provide keys during installation that encrypt {product-title} node root volumes, and enables all managed storage classes to use these keys to encrypt provisioned storage volumes. This feature was introduced in {product-title} 4.13 for Google Cloud Platform (GCP) persistent disk (PD) storage, Microsoft Azure Disk, and Amazon Web Services (AWS) Elastic Block storage (EBS), and is now supported on IBM Virtual Private Cloud (VPC) Block storage.
+
+//TODO: For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-ibm-vpc-block.adoc#byok_persistent-storage-csi-ibm-vpc-block [User-managed encryption].
+
+[id="ocp-4-15-storage-selinux-relabling-mount-options"]
+==== SELinux relabeling using mount options (Technology Preview)
+Previously, when SELinux was enabled, the persistent volume's (PV's) files were relabeled when attaching the PV to the pod, potentially causing timeouts when the PVs contained a lot of files, as well as overloading the storage backend.
+
+In {product-title} 4.15, for Container Storage Interface (CSI) driver that support this feature, the driver will mount the volume directly with the correct SELinux labels, eliminating the need to recursively relabel the volume, and pod startup can be significantly faster.
```

Comment:
I'd write specific conditions where this feature works:

This feature is Technology Preview and **all** these conditions must be satisfied to benefit from this feature:

1. PersistentVolume must have access mode ReadWriteOncePod.
2. Pod that uses the PersistentVolume must have full SELinux label specified in its spec.securityContext or spec.containers[*].securityContext. This can be easily achieved by using "restricted" SCC.
3. The CSI driver that provides the volume announces support for this feature by field `seLinuxMountSupported: true` in its CSIDriver instance. All CSI drivers that are shipped as part of {product-title} set this filed.

Reply:
Do we need to clarify that not suitable for block volume?

Reply:
I updated my comment with Filesystem volumes.

Reply:
I added list of CSI drivers that support SELinux in OCP 4.15.

Reply:
Thanks! I've added this info.